3.evaluate_model/evaluate_model.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Convert notebook to python file and execute
+# Convert notebooks to python file and execute
 jupyter nbconvert --to python \
         --FilesWriter.build_directory=scripts/nbconverted \
-        --execute evaluate_model.ipynb
+        --execute *.ipynb
```

Comment:
it is dangerous to have this wildcard here 😨 (notebooks run that you don't necessarily intend)

If there are multiple notebooks to execute, please list them explicitly. Doing so also ensures correct order of operations, if necessary.